test/integration/catalog_packages_test.go

```diff
@@ -163,12 +163,14 @@ func TestPkgCoverageDirectory(t *testing.T) {
 	for _, l := range pkg.AllLanguages {
 		definedLanguages.Add(l.String())
 	}
+	definedLanguages.Remove(string(pkg.R))
```

Comment:
nit: for consistency could you move this down below where other removals are being done? (or move those up here, either one... then they would all be done in one place which is easier to understand)